### DISCUSSION TITLE
MaxNativeZoom does not work since update to v5

### DISCUSSION THREAD

**Discussion by sequanimartin** [fleaflet/flutter_map#1624] [Q&A]
Hi,

don't know if it is an issue, so I preferred to first open a discussion here.

In my project I use this TileLayer:
`
TileLayer(
          maxNativeZoom: 18,
          urlTemplate: "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
          subdomains: ['a', 'b', 'c']);
`

as you can see I set maxNativeZoom = 18. Now in my region max zoom level is 18 and for 19/20 there are no images available. In that case by setting the maxNativeZoom to 18 it should auto-scale the image from level 18 if I zoom closer, right?

That worked perfectly in ver. 3.1.0 but now on 5.0.0 it doesn't anymore.

Suggestions?

**Comment by JaffaKetchup** [comment]
Hey @sequanimartin,

Indeed, that is the correct behaviour. I can also see that the behaviour is broken since v4, including on the latest 'master' (although the latest 'master' is slightly less broken).

I've migrated this to issue #1625.
